pom.xml

```diff
@@ -91,7 +91,7 @@
     <kafka.version>2.0.0</kafka.version>
     <confluent.version>5.3.4</confluent.version>
     <glassfish.version>2.17</glassfish.version>
-    <parquet.version>1.10.1</parquet.version>
+    <parquet.version>1.12.0</parquet.version>
```

Comment:
main thing is the parquet upgrade. Would this be compatible with Spark 2.x. guess not? I think this needs Spark 3.2? 
https://github.com/apache/spark/blob/branch-3.2/pom.xml 

this will also upgrade `parquet-avro` that we use everywhere. So just thinking about that.

Reply:
thanks, vc，I am testing...

Reply:
@liujinhui1994 : may I know whats the status on this.

Reply:
Currently working on the last problem I found. Some classes cannot be found during runtime. After this is processed, I think it should be fine. @nsivabalan